row_generation/row_generator.py

```diff
@@ -61,6 +77,8 @@ def _gen_row(self, previous_row: List[Bell], is_handstroke: bool, index: int) ->
         pass
```

Comment:
This is possibly the most important method, I'll think of something to useful to say

Reply:
Indeed.  Also, I think we should remove the underscore from the start since the linter treats it as a protected method, and doesn't recognise that when decorator classes call it that's OK (and I don't want to just disable the check on the linter).

Reply:
As in the use in `GoAndStopCallingGenerator`? I think could just be a call to `next_row`

Reply:
Yeah... I think that the current `GoAndStopCallingGenerator` implementation could be removed, and we could decide where the whole calling thing could be moved to, but that's an issue for another PR.

Reply:
For the time being, it seems that the linter doesn't require it to have a docstring because it's "private"/begins with an underscore.